microsoft-edge/devtools-guide-chromium/layers/layers-tool.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Layers tool
+description: In Microsoft Edge DevTools, the Layers tool represents the webpage in a 3D space.  The Layers tool has largely been superseded by the 3D View tool.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: devtools
+ms.date: 02/15/2022
+---
+# Layers tool
+
+The **Layers** tool represents the webpage in a 3D space.  The **Layers** tool has largely been superseded by the **3D View** tool.
```

Comment:
```suggestion
The **Layers** tool shows the different layers created by the browser rendering engine to display the final composited image of a webpage on the screen in a 3D space.  It can be used to analyze how many layers are created by your CSS code, how big they are, and how often they change. This may help you improve the rendering performance of your webpage.

The **Layers** tool is now largely superseded by the **Composited Layers** tab of the **3D View** tool.
```